dbux-data/src/Collection.js

```diff
@@ -58,26 +58,26 @@ export default class Collection {
   }
 
   /**
-   * Collections can use this to massage data after all data has been added, but before indexes have been processed.
+   * Collections can use this to resolve some additional information from raw data. This will be execute only once.
```

Comment:
The previous comment indicated the exact point in time this is called. Is that irrelevant now? Or do you want to add that part back?
I find, its always a good idea to be precise about **when** certain callbacks/hooks/API functions do what.

Also: "will be execute**d**" :)

Reply:
I add the previous comment back and keep the `This will be execute only once/behind` after them.